README.md

```diff
@@ -81,20 +147,33 @@ Suppose you're encoding geometric shapes in a Cartesian (2D) coordinate system,
 **a)**
 Write an expression for the rectangle's area.
 
+var area = (t - b) * (r - l);
+
 **b)**
 Write an expression which is true if the rectangle is taller than it is wide, and false otherwise.
 
+(t - b) > (r - l);
+
 **c)**
 Write an expression for the circumference of the biggest circle which can fit inside the rectangle.  (Hint: you'll need logic similar to that in **b**.)
 
+(t - b) > (r - l) ? 2 * 3.14 * ((r - l) * 0.5) : 2 * 3.14 * ((t - b) * 0.5); 
+
 **d)**
 Write an expression for the area of the smallest circle which completely encloses (i.e. circumscribes) the rectangle.
 
+(t - b) > (r - l) ? 2 * 3.14 * ((t - b) * 0.5) : 2 * 3.14 * ((r - l) * 0.5);
```

Comment:
3d) It's a little trickier than that, actually--this circle would clip the corners of the rectangle. In order to completely circumscribe the rectangle, you'll need a circle with a diameter equal to at least the length of a diagonal between two opposite corners of the rectangle, which is something Pythagoras is well suited to help us out with: a^2 + b^2 = c^2, where a and b are the two sides of the rectangle and c is the diagonal.